### PR TITLE
updated to API 3.0.6

### PR DESCRIPTION
**Pull request by sanalika**
gives console error when map is resetting after deathmatch or game ends, but works fine. tested.

### CODE REVIEW

File: src/hungergames/tasks/WaitingToStartTask.php [hunk 1]
```diff
@@ -56,17 +55,17 @@ public function onRun(int $tick){
                         }
                         if($count < $this->game->getMinimumPlayers()){
                                 $this->manager->setStatus("waiting");
-                                $this->HGApi->getServer()->getScheduler()->cancelTask($this->getTaskId());
+                                $this->HGApi->getScheduler()->cancelTask($this->getTaskId());
                                 $task = new WaitingForPlayersTask($this->HGApi, $this->game);
-                                $h = $this->HGApi->getServer()->getScheduler()->scheduleRepeatingTask($task, 20);
+                                $h = $this->HGAp->getScheduler()->scheduleRepeatingTask($task, 20);
```
Comment (xBeastMode) [inline src/hungergames/tasks/WaitingToStartTask.php]: 😛

File: src/hungergames/Loader.php [hunk 2]
```diff
@@ -42,7 +42,6 @@ public function onEnable(){
                 $this->scriptManager = new HGAPIScriptManager($this);
                 $this->mapBackup = new MapBackup($this);
                 $this->signManager = new SignManager($this);
-                $this->getServer()->getCommandMap()->register("hg", new HGCommand($this));
```
Comment (xBeastMode) [inline src/hungergames/Loader.php]: this is for the `/hg` command in game, without it, it won't work.

File: src/hungergames/tasks/LoadGamesTask.php [hunk 3]
```diff
@@ -4,6 +4,8 @@
 use pocketmine\scheduler\Task;
 use pocketmine\utils\TextFormat;
 
+use pocketmine\Server;
```
Comment (xBeastMode) [inline src/hungergames/tasks/LoadGamesTask.php]: what's this supposed to fix?

**Review by xBeastMode — CHANGES_REQUESTED** [review]

File: src/hungergames/tasks/WaitingToStartTask.php (hunk 1 above)
Comment (sanalika) [reply src/hungergames/tasks/WaitingToStartTask.php]: im an idiot :D thx for correction :D

File: src/hungergames/Loader.php (hunk 2 above)
Comment (sanalika) [reply src/hungergames/Loader.php]: my stupid bugfixes :D thx for correction :D

**Comment by sanalika** [comment]
i've edited mistakes :D sorry for my stupidity :( i was editing the code in npp and copying steps to github :/

File: src/hungergames/tasks/LoadGamesTask.php (hunk 3 above)
Comment (sanalika) [reply src/hungergames/tasks/LoadGamesTask.php]: idk but i tried something, it was giving error to my console

Comment (sanalika) [reply src/hungergames/tasks/LoadGamesTask.php]: now, it doesn't give any error. but i think it didn't belong to it xd may it be? idk xd

**Comment by xBeastMode** [comment]
Thanks, I'll check it one of these days.

**Comment by xBeastMode** [comment]
I had already updated, and just pushed the changes now, but thanks! 👍 Don't doubt to make any other pull requests 👍

**Pull request closed without merging**